airbyte-integrations/connectors/source-linnworks/metadata.yaml

```diff
@@ -16,15 +16,15 @@ data:
   name: Linnworks
   registries:
     cloud:
-      enabled: true
+      enabled: false
     oss:
-      enabled: true
+      enabled: false
```

Comment:
Of note - Linworks was hidden but still in the OSS and Cloud registries.  I think this was a mistake, given the LaunchDarkly feature flag?

Reply:
@evantahler I believe @ChristoGrab updated this connector to the latest CDK so it does not use legacy state.
I think you can revert your changes on this metadata.yaml

Reply:
https://github.com/airbytehq/airbyte/pull/35557

Reply:
@ChristoGrab can you remove linnworks feature flag if you think it can now work on Cloud?

Reply:
https://github.com/airbytehq/airbyte/pull/35752/commits/dfefdd875e4fe903cb270fb1f6ead3893b0a4a0d